package.json

```diff
@@ -92,7 +92,7 @@
     "react-addons-shallow-compare": "15.3.0",
     "react-addons-update": "15.3.0",
     "react-click-outside": "2.1.0",
-    "react-day-picker": "1.3.2",
+    "react-day-picker": "2.4.1",
```

Comment:
@retrofox FIY I've run `make shrinkwrap` and it updated a bunch of dependencies that I guess are related to `react-day-picker`.
After cleaning and reinstalling, (to my knowledge) Calypso seems to work as expected.